### DISCUSSION TITLE
Delete File After Download

### DISCUSSION THREAD

**Discussion by codecubelabs** [drakkan/sftpgo#1689] [Q&A]
Hi, Is it possible to implement a policy or event to delete the file(s) after they were downloaded or accessed in an SFTPGo folder?

**Comment by peteraramaldes** [comment]
@drakkan 
Could you help on how to configure this action? I want to delete the file after sucessfully downloaded.

**Reply by marziaelia** [reply]
> @drakkan Could you help on how to configure this action? I want to delete the file after sucessfully downloaded.

Hi,

For guaranteed support or faster response times, we recommend considering our [commercial support](https://sftpgo.com).

Also, we kindly ask not to tag maintainers directly unless previously requested—thank you for your understanding!

Regarding your request: SFTPGo can delete files after a successful download using the [EventManager](https://docs.sftpgo.com/2.6/eventmanager/) or a [custom hook](https://docs.sftpgo.com/2.6/custom-actions/). 
The best approach depends on your specific use case and environment.

We hope the documentation links provided will help you resolve your issue. 
If you need further assistance, feel free to ask the community.
